src/index.css

```diff
@@ -18,6 +13,7 @@ ul {
 a {
   text-decoration: none;
   color: inherit;
+  display: block;
```

Comment:
please don't add the display-block property here. The link can be faced in the text. A link is an Inline tag.